spec/inject_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'open_array'
+
+describe Array do
+
+  describe '#my_inject' do
+
+    it 'sums numbers' do
+      expect([5,5,5].my_inject{ |sum, next_number | sum + next_number }).to eq(15)
+    end
+
+    it 'sums more numbers' do
+      expect([2,3,4].my_inject{ |sum, next_number | sum + next_number }).to eq(9)
```

Comment:
Line is too long. [81/80]<br>Space missing after comma.<br>Space after last block parameter detected.<br>Space missing to the left of {.